entries/m/microworkers.com.json

```diff
@@ -0,0 +1,14 @@
+{
+  "Microworkers": {
+    "domain": "microworkers.com",
+    "url": "https://www.microworkers.com",
```

Comment:
```suggestion
```

The `url` is not needed when going to `https://[domain]` works.

Reply:
Thank you for your reply. This is my first submission, so I don’t quite understand some of the rules. I will pay more attention to it later!